### ISSUE TITLE
XY plot to simultaneously vary LoRA number and strength?

### ISSUE THREAD

**Issue by kasukanra** [LucianoCirino/efficiency-nodes-comfyui#70]
I recently started using this plugin, and I was able to use the manual XY entry node to achieve individually iterating over LoRAs with incremental epoch names or varying the strength of a single LoRA. 

![prodigy_02](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/26351130/c189583a-4688-4c33-a8bf-895087eb0017)
![prodigy_01](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/26351130/edeaf564-5a0e-4e1e-bb96-d54942a01b07)

However, I was wondering if it was possible to replicate this workflow from AUTOMATIC1111 xyz scripts.

![functionality_01](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/26351130/87d50a37-e9b8-45b2-a9ae-6f00fbbb42c3)

Basically, I would be combining both manual XY entries so that I can see the variation in LoRA epoch and LoRA strength. In A1111, I would use Prompt S/R to replace both the epoch number and the LoRA strength, but I'm not sure how to achieve that using efficiency nodes.

**Comment by wouterverweirder** [comment]
Also interested in this feature.

**Comment by LucianoCirino** [comment]
Interesting. Will look into a good method.

**Comment by andreszs** [comment]
Any news on this? Is there no other way to create a grid to compare the same lora(s) but with different strengths?

**Comment by LucianoCirino** [comment]
> Any news on this? Is there no other way to create a grid to compare the same lora(s) but with different strengths?

I apologize for the delay, I am working on a big SDXL update. I will try to capture as many of these pending issues/items as possible.

**Comment by LucianoCirino** [comment]
This is now possible, check out the "XY Input: LoRA Plot" node under input_mode "X: LoRA Batch, Y: Model Strength"

**Comment by andreszs** [comment]
Thanks @LucianoCirino , would you be so kind to also update the README in order to add a simple example of using this LoRA Plot node to compare different strength values?

**Comment by LucianoCirino** [comment]
> Thanks @LucianoCirino , would you be so kind to also update the README in order to add a simple example of using this LoRA Plot node to compare different strength values?

Simply added a new XY PLot image to the README

![image](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/112517630/a563bfd4-c2d7-4c74-b6ac-f8a17fb1dd31)

**Comment by andreszs** [comment]
@LucianoCirino, for some reason I'm getting this error with the **XY Input Lora Plot** node:

> Error occurred when executing XY Input: LoRA Plot:
> 
> TSC_XYplot_LoRA_Plot.xy_value() missing 1 required positional argument: 'X_subdirectories'
> 
>   File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 144, in recursive_execute
>     output_data, output_ui = get_output_data(obj, input_data_all)
>   File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in get_output_data
>     return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
>   File "C:\ComfyUI_windows_portable\ComfyUI\execution.py", line 67, in map_node_over_list
>     results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))

### Workflow
![workflow-1](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/10298071/b0ff5892-e3f2-4c33-8174-f904c385c8b0)

I've downloaded the complete project's ZIP and replaced the old efficiency-nodes-comfyui contents, then restarted UI.

**Comment by LucianoCirino** [comment]
@andreszs try deleting the node and placing a new one. There should not be two lora_stack inputs.

**Comment by andreszs** [comment]
I tried that but there is a **_X_subdirectories_** input that cannot be connected to a primitive and it can't be converted to widget either, so it just remains there disconnected.

![image](https://github.com/LucianoCirino/efficiency-nodes-comfyui/assets/10298071/8c524702-3b16-4d31-8339-1499a001fedf)
